decomp/semantics/uds.py

```diff
@@ -49,7 +59,8 @@ class UDSCorpus(PredPattCorpus):
 
     def __init__(self,
                  graphs: Optional[PredPattCorpus] = None,
-                 annotations: List['UDSDataset'] = [],
+                 annotations: List['UDSAnnotation'] = [],
+                 raw_properties: Set[str] = set(),
```

Comment:
Is there a reason to allow the user to set the `raw_properties`? Should this be implicitly in the subclass of `UDSAnnotation`?